### PR TITLE
add unit test for generatePIN

### PR DESCRIPTION
**Pull request by diamemorreau**
Hi! 👋

I have created an unit test for the generatePIN method. 

This is my first ever contribution to an open source project. Could you provide me some feedback? And I would like to write the unit tests for other methods as well. 

Screenshot 2023-04-23 at 15 05 59

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
```
Comment (muke1908) [inline package.json:20]: This should go to `devDependencies`.

**Comment by muke1908** [comment]
@diamemorreau Thanks. Left a comment.

**Review by muke1908 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (diamemorreau) [reply package.json]: Thanks, I have moved the dependencies.

**Comment by muke1908** [comment]
closes #271

**Pull request merged**